load-test/send_load.sh

```diff
@@ -22,15 +22,59 @@ then
     PORT=16657
 fi
 
+balance_query() {
+    balance=$("${DAEMON}" q bank balances $1 --node $RPC --output json)
+    balanceres=$(echo "${balance}" | jq -r '.balances')
+    return "$balanceres"
+}
+
 RPC="http://${IP}:${PORT}"
+num_txs=35
+num_msgs=30
 acc1=$($DAEMON keys show account$FROM -a --home $DAEMON_HOME-1 --keyring-backend test)
 acc2=$($DAEMON keys show account$TO -a --home $DAEMON_HOME-1 --keyring-backend test)
-seq1=$("${DAEMON}" q account "${acc1}" --node $RPC --output json)
-seq1no=$(echo "${seq1}" | jq -r '.sequence')
-bound=`expr 10000 + $seq1no`
-for (( a=$seq1no; a<$bound; a++ ))
+echo "** Balance of Account 1 before send_load :: **"
+balance_query "$acc1"
+echo "** Balance of Account 2 before send_load :: **"
+balance_query "$acc2"
+cd ~/
+for (( a=0; a<$num_txs; a++ ))
 do
-    sTx=$("${DAEMON}" tx bank send "${acc1}" "${acc2}" 1000000"${DENOM}" --chain-id "${CHAINID}" --keyring-backend test --home $DAEMON_HOME-1 --node $RPC --output json -y --sequence $a) 
-    sTxHash=$(echo "${sTx}" | jq -r '.txhash')
-    echo "** TX HASH :: $sTxHash **"
+		unsignedTx=$("${DAEMON}" tx bank send "${acc1}" "${acc2}" 1000000"${DENOM}" --chain-id "${CHAINID}" --output json --generate-only --gas 500000 > unsigned.json)
+		unsignedTxres=$(echo "${unsignedTx}")
+		for (( b=0; b<$num_msgs; b++))
+		do
+    		cat unsigned.json | jq '.body.messages |= . + [.[-1]]' > unsigned.json.bk
+            mv unsigned.json.bk unsigned.json 
+		done
+		seq1=$("${DAEMON}" q account "${acc1}" --node $RPC --output json)
+		seq1no=$(echo "${seq1}" | jq -r '.sequence')
```

Comment:
The goal of this test is to really run tx in parallel

Why not using this solution for managing sequence numbers: https://github.com/cosmos/qa-integration/pull/12#discussion_r789139979

Reply:
@kaustubhkapatral let's query for sequence numbers before the for-loop and just recalculate here instead of querying store